developers/vscode/README.md

```diff
@@ -0,0 +1,214 @@
+# Ocean Protocol VSCode Extension
+
+The Ocean Protocol VSCode extension brings Ocean Protocol's powerful data sharing and monetization capabilities directly into your development environment. Whether you're publishing datasets, running compute-to-data jobs, or managing Ocean Protocol assets, you can do it all without leaving Visual Studio Code.
+
+## Getting Started
+
+The extension adds a dedicated Ocean Protocol section to your VSCode workspace. You'll find it in the activity bar on the left side of your VSCode window, marked with the Ocean Protocol logo.
+
+### First-time Setup
+
+When you first open the extension, you'll need to configure a few settings:
+
+1. RPC URL - This is your connection to the blockchain
+2. Ocean Node URL - Where your Ocean node is running
+3. Private Key - For signing transactions (keep this secure!)
+
+{% hint style="info" %}
+Your private key is stored securely within VSCode and is never transmitted to external services.
+{% endhint %}
+
+## Key Features
+
+### Publishing Data Assets
+
+Turn your data into a monetizable asset on Ocean Protocol with just a few clicks.
+
+1. Select "Publish Asset" from the Ocean Protocol panel
+2. Choose your metadata file (JSON format)
+3. Click "Publish Asset"
+
+The extension handles all the complex interactions with Ocean Protocol, including:
+- Creating the necessary NFTs and datatokens
+- Encrypting your data URLs
+- Setting up access control
+- Publishing metadata to the network
+
+{% hint style="success" %}
+You can track the publication progress directly in VSCode's output panel.
+{% endhint %}
+
+### Asset Discovery and Management
+
+Easily explore and manage Ocean Protocol assets:
+
+- Search for assets using their DID
+- View detailed asset information
+- See pricing and access details
+- Monitor asset status
+
+### Compute-to-Data Operations
+
+Run privacy-preserving computations on data without exposing the raw data:
+
+1. Choose your dataset
+2. Select your algorithm
+3. Configure compute settings
+4. Start your compute job
+
+Track your compute jobs in real-time:
+- View job status and progress
+- Access computation logs
+- Download results when complete
+
+### P2P Network Interaction
+
+Monitor and interact with Ocean Protocol's P2P network directly from VSCode:
+
+- View your node's ID
+- See connected peers
+- Monitor network health
+- Track node status
+
+### Asset Downloads
```

Comment:
We said that download and publish features for later iterations. For first release we have c2d features. Can we mention this somewhere?

Reply:
Yeah, you're right, I'll update this